### PR TITLE
Add PokeWalker encounters

### PR DESCRIPTION
**Pull request by wwwwwwzx**

### CODE REVIEW

**Comment by wwwwwwzx** [comment]
I wonder if it's better to use reference to mark the location instead of using the clone method..

**Comment by javierhimura** [comment]
Is better to mark the location in the proper static object like any other encounters only possible in one location and leave clone method only for static locations with more that one met location (roaming only i think)

**Comment by javierhimura** [comment]
Also Gift = true should be added to pokewalker encounters, to mark illegal pokewalker pokemons that are not in pokeball

**Comment by wwwwwwzx** [comment]
I guess so. Thank you

File: PKHeX/Resources/text/zh/LegalityCheckStrings_zh.txt [hunk 1]
```diff
@@ -145,7 +145,6 @@ V111 = 非帮手宝可梦拥有隐藏特性。
 V300 = 在非群战/朋友原野区获得隐藏特性。
 V112 = 隐藏特性不可获得。
 V217 = 隐藏洞穴捕获的宝可梦必定为隐藏特性。
-V218 = 在野外普通相遇不可能有隐藏特性。
```
Comment (evandixon) [inline PKHeX/Resources/text/zh/LegalityCheckStrings_zh.txt]: Why did you remove this line?

Comment (kwsch) [inline PKHeX/Resources/text/zh/LegalityCheckStrings_zh.txt]: Doesn't currently exist in LegalityCheckStrings.cs :)
